specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2019-07-01/databoxedge.json

```
@@ -0,0 +1,4125 @@
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/{deviceName}/updateSummary/default": {
+      "get": {
+        "tags": [ "Devices" ],
+        "summary": "Gets information about the availability of updates based on the last scan of the device. It also gets information about any ongoing download or install jobs on the device.",
+        "operationId": "Devices_GetUpdateSummary",
+        "parameters": [
+          {
+            "name": "deviceName",
+            "in": "path",
+            "description": "The device name.",
+            "required": true,
+            "type": "string"
+          },
+          { "$ref": "#/parameters/subscriptionIdParameter" },
+          { "$ref": "#/parameters/resourceGroupNameParameter" },
+          { "$ref": "#/parameters/apiVersionParameter" }
+        ],
+        "responses": {
+          "200": {
+            "description": "The update summary.",
+            "schema": { "$ref": "#/definitions/UpdateSummary" }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": { "$ref": "#/definitions/CloudError" }
+          }
+        },
+        "x-ms-examples": {
+          "UpdateSummaryGet": {
+            "$ref": "./examples/UpdateSummaryGet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataBoxEdge/dataBoxEdgeDevices/{deviceName}/uploadCertificate": {
+      "post": {
+        "tags": [ "Devices" ],
+        "description": "Uploads registration certificate for the device.",
+        "operationId": "Devices_UploadCertificate",
+        "parameters": [
+          {
+            "name": "deviceName",
+            "in": "path",
+            "description": "The device name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The upload certificate request.",
+            "required": true,
+            "schema": { "$ref": "#/definitions/UploadCertificateRequest" }
+          },
+          { "$ref": "#/parameters/subscriptionIdParameter" },
+          { "$ref": "#/parameters/resourceGroupNameParameter" },
+          { "$ref": "#/parameters/apiVersionParameter" }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully uploaded the registration certificate.",
+            "schema": { "$ref": "#/definitions/UploadCertificateResponse" }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": { "$ref": "#/definitions/CloudError" }
+          }
+        },
+        "x-ms-examples": {
+          "UploadCertificatePost": {
+            "$ref": "./examples/UploadCertificatePost.json"
+          }
+        }
+    
```

Fulfilment -> Fulfillment
```suggestion
          "enum": [ "Untracked", "AwaitingFulfillment", "AwaitingPreparation", "AwaitingShipment", "Shipped", "Arriving", "Delivered", "ReplacementRequested", "LostDevice", "Declined", "ReturnInitiated", "AwaitingReturnShipment", "ShippedBack", "CollectedAtMicrosoft" ],
```